el8/Dockerfile.buildtime

```diff
@@ -2,23 +2,34 @@ FROM @BASE_IMAGE_NAME@
 LABEL maintainer="CMS Build"
 LABEL name="CMS Worker Node on EL - Buildtime"
 
-RUN dnf install --disablerepo=epel -y perl-Carp perl-PathTools perl-Data-Dumper perl-Errno perl-Exporter perl-File-Path \
-          perl-File-Temp perl-Getopt-Long perl-IO perl-Text-ParseWords perl-constant perl-Digest-MD5 \
-          perl-PathTools perl-Data-Dumper perl-Digest-MD5 perl-Exporter perl-File-Path perl-File-Temp \
-          perl-Getopt-Long libdrm perl-File-Which perl-Scalar-List-Utils perl-Term-ANSIColor perl-CGI \
-          perl-DBI perl-Encode perl-Memoize perl-Storable perl-Time-HiRes perl-Time-Local perl-YAML \
-          perl-IO-Compress &&\
-    dnf install --disablerepo=epel -y rpm-build git nspr nss rsync wget openssl-devel ncurses-devel \
-          libX11-devel readline-devel tcl-devel tk-devel mesa-libGLU-devel libXext-devel libXft-devel \
-          libXpm-devel java-1.8.0-openjdk-devel java-11-openjdk-devel libtool gcc-c++ &&\
-    dnf --enablerepo=powertools install -y texinfo &&\
-    epel_pkgs="@EPEL_PACKAGES@" &&\
-    if [ -n "$epel_pkgs" ] ; then dnf install -y $epel_pkgs; fi &&\
-    xcmd="@EXTRA_COMMAND@" &&\
-    if [ "$xcmd" = "" ] ; then xcmd=true; fi &&\
-    eval $xcmd &&\
-    dnf update -y ca-certificates &&\
-    dnf clean all &&\
-    echo "Timestamp: @BUILD_DATE@" > /image-build-info.txt
+RUN dnf install --disablerepo=epel -y perl-Carp perl-PathTools perl-Data-Dumper perl-Errno perl-Exporter \
+	perl-File-Path perl-File-Temp perl-Getopt-Long perl-IO perl-Text-ParseWords perl-constant \
+	perl-Digest-MD5 perl-PathTools perl-Data-Dumper perl-Digest-MD5 perl-Exporter perl-File-Path \
+	perl-File-Temp perl-Getopt-Long perl-File-Which perl-Scalar-List-Utils perl-Term-ANSIColor \
+	perl-CGI perl-DBI perl-Encode perl-Memoize perl-Storable perl-Time-HiRes perl-Time-Local \
+	perl-YAML perl-IO-Compress \
```

Comment:
same here?

Reply:
by the way you need to run `dnf clean all` after every `dnf` command otherwise every layer will contain the dnf cache